docs/webhooks.md

```diff
@@ -0,0 +1,72 @@
+# Webhooks
+
+?> Available `5.2.0`
```

Comment:
perhaps it makes sense to have a global "turn on webhooks" - so if I replicate a server for development / testing environment, I can turn all the webhooks off at once?